buildspecs/buildspec-unittest-java8.yml

```diff
@@ -0,0 +1,17 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: openjdk8
```

Comment:
there are two java8:
- java8
- java8-al2

So I assume java8-al2 use our coretto because openjdk is not available on our AL2 images.

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/3804518/101535665-d4c6de80-394d-11eb-847c-6c7da4054f24.png">